src/aggregators/aggregators.jl

```diff
@@ -163,8 +163,9 @@ algorithm with optimal binning,  Journal of Chemical Physics 143, 074108
 """
 struct DirectCRDirect <: AbstractAggregatorAlgorithm end
 
-const JUMP_AGGREGATORS = (Direct(), DirectFW(), DirectCR(), SortingDirect(), RSSA(), FRM(),
-                          FRMFW(), NRM(), RSSACR(), RDirect(), Coevolve())
+const JUMP_AGGREGATORS = (Direct(), DirectFW(), DirectCR(), SortingDirect(), RSSA(),
+    FRM(),
+    FRMFW(), NRM(), RSSACR(), RDirect(), Coevolve())
```

Comment:
so with a lot of these it's because it's fighting with join_lines_based_on_source=true

on this line in particular

`const JUMP_AGGREGATORS = (Direct(), DirectFW(), DirectCR(), SortingDirect(), RSSA(),
    FRM(), FRMFW(), NRM(), RSSACR(), RDirect(), Coevolve())
`

would be output with =false

in the original source FRM() has a newline after it which is why it's kept. So I have to make the algo aware of that.

But I'd be curious to see what the format would be like in you do join_lines_based_on_source=false at least for the initial format and then once you get a split you're ok with turn it back on or something